### PR TITLE
Client readme

### PR DESCRIPTION
**Pull request by corylanou**
Update the client readme.  Add missing comments.  Fix inconsistencies in code based on desired behavior.

### CODE REVIEW

**Comment by otoolep** [comment]
The changes to the README. Wouldn't it better if all that good information were in the form of GoDoc comments? That way we can just generate documentation *automatically*.

**Comment by corylanou** [comment]
@otoolep this was generated from the godoc comments.  I agree that we need to update our documents as well to give a much better version of this.  When that is done, we should remove it from here.  For now, we have nothing, so this was just a first attempt at getting something.

**Comment by otoolep** [comment]
I was thinking it was very similar to GoDoc. :-)

Perhaps I am missing something, but it's not clear to me why GoDoc-generated stuff needs to be added to the README. Why not just let the source comments speak for themselves, and simply do what Bolt does in its README? I.e. reference the equivalent of:

https://godoc.org/github.com/boltdb/bolt

And use the README for purely some examples, again like Bolt does.

File: cmd/influxd/server_integration_test.go [hunk 1]
```diff
@@ -726,7 +726,7 @@ func TestClientLibrary(t *testing.T) {
 		}
 
 		if test.query.Command != "" {
-			time.Sleep(50 * time.Millisecond)
+			time.Sleep(100 * time.Millisecond)
```
Comment (otoolep) [inline cmd/influxd/server_integration_test.go]: Did you find you needed to do this?

Comment (corylanou) [inline cmd/influxd/server_integration_test.go]: Yes, 50 was sometimes coming up short.  100 always worked.  Until we get our wait endpoint back up to par this feels like the best we can do.

**Comment by corylanou** [comment]
@otoolep Sometimes I work to hard :-).  I forgot that godoc is automatic on that site.

File: cmd/influxd/server_integration_test.go (hunk 1 above)
Comment (otoolep) [inline cmd/influxd/server_integration_test.go]: OK, it's just a stop-gap anyway so cool.

**Comment by otoolep** [comment]
Latest changes look find to me. +1.

**Pull request merged**